articles/extensions/custom-social-extensions.md

```diff
@@ -61,6 +62,15 @@ Using the slider, enable this social connection for the apps that you want to us
 
 Once you have enabled/disabled the appropriate apps, click **Save**.
 

```

Comment:
The customer also commented on this: `It's lacking information on what programming language to use in the fetch userprofile function, what methods are available to that function.
Furthermore it lacks info on how to map fields from the 3rd party API to the ones of Auth0.,`

The language is JS (not sure if we can use another). You could clarify this under paragraph `#### New Connection: Settings`. I don't know how the map fields works but sandrino would know.